templates/docs/patterns/search-box.md

```diff
@@ -47,6 +47,29 @@ When search box is expanded the overlay element (`p-navigation__search-overlay`)
 
 <div class="embedded-example"><a href="/docs/examples/patterns/navigation/search-light" class="js-example"> View example of the search navigation </a></div>
 
+## Accessibility
+
+{# copy doc: https://docs.google.com/document/d/1YH05QhjuXLl93pdh_9ETYhalY7lU7tfNQ5EKD4ZRfw0/edit# #}
```

Comment:
Commented on those in other PRs as well - I feel like we don't need these links (as we should not treat these documents as copy docs, any further edits should happen in the code).

Reply:
Ok no probs I'll remove them from all